membership-common/src/test/scala/com/gu/memsub/subsv2/services/SubscriptionServiceTest.scala

```diff
@@ -262,6 +265,13 @@ class SubscriptionServiceTest extends Specification {
       GetCurrentPlans.currentPlans(digipackSub, referenceDate, catalog).map(_.head.id.get) mustEqual \/-("idDigipack")
     }
 
+    "if you cancel an Ad-Lite in the trial period, tell you you're not a subscriber" in {
+      val now = new LocalDate(2025, 2, 13) // still in trial
+      val adLite = subscriptionReads.reads(Json.parse(adLiteCancelledInTrial)).get
```

Comment:
the rest of the tests are using minimal data structures defined in scala, but the fashion now is to use raw data pasted from zuora responses, so I've done that instead.